<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shopPrice</class>
 <widget class="QWidget" name="shopPrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_money">
       <property name="text">
        <string>金额：</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_money</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_money">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_strategy">
       <property name="text">
        <string>策略：</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_strategy</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_strategy">
       <item>
        <property name="text">
         <string>正常收费</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>满300返100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>打八折</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_result">
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
